test/postgresql.discover.test.js

```diff
@@ -257,6 +288,33 @@ describe('Discover model primary keys', function() {
     });
   });
 
+  it('primary key should be discovered, and db generates instances properly', function(done) {
+    db.discoverPrimaryKeys('demo', function(err, models) {
+      if (err) {
+        console.error(err);
+        done(err);
+      } else {
+        models.forEach(function(m) {
```

Comment:
Why loop if there is only 1 model?

Reply:
Match with other tests.

Reply:
I think it makes sense to not loop if it's only one model, even if it's inconsistent with the other tests, but it's not a big deal.

Reply:
edited, thanks!